src/EventCalendar.jsx

```diff
@@ -0,0 +1,18 @@
+import { useEffect, useRef } from "react";
+import { EventCalendar } from "@dhx/trial-eventcalendar";
+import "@dhx/trial-eventcalendar/dist/event-calendar.css";
+
+export default function CalendarView(props) {
+	let container = useRef();
+
+	useEffect(() => {
+		new EventCalendar(container.current, {
+			events: props.events,
+			date: props.date,
+		});
+
+		return () => (container.current.innerHTML = "");
```

Comment:
need to do `eventCalendar.destructor()`  on unmount https://docs.dhtmlx.com/eventcalendar/api/methods/js_eventcalendar_destructor_method/